integration-tests/realm-apps/with-db-flx/services/mongodb/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "mongodb",
+  "name": "mongo-db",
```

Comment:
It was so that we'd use the [default service name](https://www.mongodb.com/docs/atlas/app-services/functions/context/#std-label-context-services) for mongo clients. But, I see now that the default is actually `mongodb-atlas`. So if you think it's fine for us to change from `mongodb` I can go ahead and change to `mongodb-atlas`. This will also be the default argument we're using when users call `mongoClient()`.

Reply:
> the default is actually `mongodb-atlas` -- good find 🤔 I do remember seeing that before and I wonder why that was not the default on realm-web then.

Reply:
Update: I've reverted this back to `mongodb` and also changed `mongoClient()` to *not* have a default argument, but rather let the user be explicit about which service to connect to.

> ..the default on realm-web

P.S. Even though realm-web's internal helper function has a default argument, it doesn't seem like the public facing method actually documents that a default argument exists. Not in the docs and not by the TS type:
```ts
// Public facing
public mongoClient(serviceName: string): Realm.Services.MongoDB {
  return createMongoDBRemoteService(this.fetcher, serviceName);
}

// Helper
export function createService(fetcher: Fetcher, serviceName = "mongo-db"): MongoDBService {
  return { db: createDatabase.bind(null, fetcher, serviceName) };
}
```